src/plays/git-hub-profile-search/Readme.md

```diff
@@ -0,0 +1,27 @@
+# GitHub Profile Search
+
+A play in which when you type someone's github username it displays there public github profile.
+
```

Comment:
This play can be developed without Context API as well. You can explain why you have used Context API, learning, etc.

Reply:
added the reason

Reply:
Nice.